README.md

```diff
@@ -1,3 +1,10 @@
+# This project is DEPRECATED
+
+With the release of Kotlin 1.7.20-Beta (https://blog.jetbrains.com/kotlin/2022/08/kotlin-1-7-20-beta/) and the new 
+Memory Model for Kotlin Native becoming the default, there's no longer a need for this library. Version 0.9.0 helps 
+transition over to the new memory model without having to make any changes related to CoroutineWorker code. The latest 
+version of CoroutineWorker uses the same implementation for all platforms.
+
```

Comment:
Should we maybe just leave the documentation below set to the last version that did things differently, and tell people to just use the latest version only if they are using the new memory model?

Reply:
Wait why are we doing a 0.9.0 again? I thought we were gonna do 1.0